src/uu/pinky/pinky.md

```diff
@@ -4,4 +4,11 @@
 pinky [OPTION]... [USER]...
 ```
 
-Displays brief user information on Unix-based systems
+Displays brief user information on Unix-based systems.
+
+## Note
+
+When built with musl libc, the `pinky` utility may show incomplete or
+missing user information due to musl's stub implementation of `utmpx`
+functions. This limitation affects the ability to retrieve logged-in
+user details.
```

Comment:
I don't think this is working like you want it to. This section is ignored by uudoc and by the help parser. We could merge this and fix those somehow, but it's something to keep in mind. Also, while this is nice for the online docs, I think this shouldn't show up in help strings when uutils is not compiled for musl.

Reply:
I have changed it to be shown conditionally by including some `cfg` logic in the rust files. My reasoning for not doing initially was precisely to include this warning about musl in generated documentation using the .md files.

Reply:
And I do like that idea! I was just saying that due to the way we process these files, I don't think this paragraph would show up in the documentation, so it wasn't working like you intended to.